### DISCUSSION TITLE
What does the nvdash symbol mean or words?

### DISCUSSION THREAD

**Discussion by AddictArts** [NvChad/NvChad#2755] [Q&A] [answered]
I looked but could not find any description, thanks for the help.

                                                                                        ▄ ▄                     
                                                                                    ▄   ▄▄▄     ▄ ▄▄▄ ▄ ▄       
                                                                                    █ ▄ █▄█ ▄▄▄ █ █▄█ █ █       
                                                                                 ▄▄ █▄█▄▄▄█ █▄█▄█▄▄█▄▄█ █       
                                                                               ▄ █▄▄█ ▄ ▄▄ ▄█ ▄▄▄▄▄▄▄▄▄▄▄▄▄▄    
                                                                               █▄▄▄▄ ▄▄▄ █ ▄ ▄▄▄ ▄ ▄▄▄ ▄ ▄ █ ▄  
                                                                             ▄ █ █▄█ █▄█ █ █ █▄█ █ █▄█ ▄▄▄ █ █  
                                                                             █▄█ ▄ █▄▄█▄▄█ █ ▄▄█ █ ▄ █ █▄█▄█ █  
                                                                                 █▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄█ █▄█▄▄▄█

**Comment by siduck** [comment, marked as answer]
@AddictArts its in arabic kufic script which means knowledge comes before speech and action

**Reply by ahmedelgabri** [reply]
@siduck I'm curious to know how it was created/generated?

**Reply by siduck** [reply]
@ahmedelgabri  my friend @xyl1t did it :heart:

**Reply by xyl1t** [reply]
As salamu aleikum, thanks for the mention!

My first step was to make the initial design in paint where I could edit pixels by hand (because doing it in text right away is very tedious). The design was iterated over multiple times until the design was satisfactory.
After that step I transferred the image to text using Neovim and utilising just these characters:

- Half block: `▄`
- Full block: `█`

That's it! I don't have any special software for this process, but I imagine one could write a simple script to convert the pixels of an image to text.

**Reply by ahmedelgabri** [reply]
Wa Alaykom AlSalam @xyl1t, Mashallah great stuff! Thanks for the information, very useful.

**Reply by TalhaZaheer1** [reply]
Just discovered this, new to neovim and cant even install an lsp properly yet, just spent last 15 min understanding this text.

**Reply by siduck** [reply]
@TalhaZaheer1  brother read nvchad.com, just configure your lsp and run MasonInstallAll, it'll install the lsp binary for you

**Reply by AddictArts** [reply]
> Just discovered this, new to neovim and cant even install an lsp properly yet, just spent last 15 min understanding this text.

Been there! I had similar issues. While NvChad helped to get into neovim I ended up leaving it, due to these issues and too opinionated.

I found LavyVim to be a far better and easier to grasp config, plus good docs. I’d give it a try.

**Reply by TalhaZaheer1** [reply]
Im taking it slow with NvChad, too lazy to change to lazy.vim now. Chatgpt is helping alot.

**Reply by AddictArts** [reply]
> Im taking it slow with NvChad, too lazy to change to lazy.vim now. Chatgpt is helping alot.

Cool, best wishes. The Neovim kickstart project is very helpful to learn as well if you run into any trouble.
